### ISSUE TITLE
PHP gettext support broken

### ISSUE THREAD

**Issue by kampalex** [heroku/heroku-buildpack-locale#8]
Because of Heroku DevCenter recommodation I'm using this buildpack to install additional locales I need in our Node/PHP project.

I already created a file .locales which contains the locale names used on Cedar-14:
```
nl_NL.utf8
en_GB.utf8
de_DE.utf8
```

No LANG config/environment variable has been set - according to 'deprecated' warning (anyway, it shows up always). I've already submit feedback to Heroku in order to correct the information shown [here](https://devcenter.heroku.com/articles/heroku-16-stack#locales).

In build log installs are executed:
```
-----> Locale app detected
=!= Support for the '$LANG' config variable to install locales will be removed soon.
=!= You should switch to a '.locales' file.
=!= See https://github.com/heroku/heroku-buildpack-locale
=== Updating or installing language-pack-nl-base
[...]
The following additional packages will be installed:
  language-pack-nl
The following NEW packages will be installed:
  language-pack-nl language-pack-nl-base
0 upgraded, 2 newly installed, 0 to remove and 69 not upgraded.
Need to get 0 B/2,039 kB of archives.
After this operation, 8,582 kB of additional disk space will be used.
Download complete and in download only mode
=!= Language is en. Nothing to install.
=== Updating or installing language-pack-de-base
[...]
The following additional packages will be installed:
  language-pack-de
The following NEW packages will be installed:
  language-pack-de language-pack-de-base
0 upgraded, 2 newly installed, 0 to remove and 69 not upgraded.
Need to get 0 B/3,104 kB of archives.
After this operation, 13.4 MB of additional disk space will be used.
Download complete and in download only mode
=== Unpacking and configuring all locales
-----> Node.js app detected
[...]
-----> Build succeeded!
-----> PHP app detected
-----> Bootstrapping...
-----> Installing platform packages...
       - php (7.1.10)
[...]
          - ext-gettext (bundled with php)
[...]
       Released [...]
```

It turns out that Dutch was loaded properly via PHP/gettext, but German/English wasn't. By analyzing the compile script, supplying the above locales are allowed/expected. Because of this strange situation I've compared the output of gettext before and after adding buildpack. I've ran setlocale functions in PHP.

Without buildpack:
setLocale(LC_MESSAGES,[nl_NL / nl_NL.utf8 / de_DE / de_DE.utf8]) = false
setLocale(LC_MESSAGES,[en_GB / en_GB.utf8]) = echoes the input locale + applies translations

Using buildpack:
setLocale(LC_MESSAGES,[nl_NL, nl_NL.utf8]) = echoes the input locale + applies translations
setLocale(LC_MESSAGES,[en_GB / en_GB.utf8 / de_DE / de_DE.utf8]) = false

Another strange effect is the error message after `heroku run bash`. It's only after applying the buildpack:
```
character map file `utf8en_US' not found: No such file or directory
no output file produced because warnings were issued
```

As you can see this buildpack installs Dutch and activates it, but also breaks things.
Anyone else having this issues? Because of this I'm playing safe and stay at Cedar-14 until it's working properly.

**Comment by dmathieu** [comment]
Could you open a support ticket on help.heroku.com, referencing the app this is happening on?
We'll then be able to better help you by investigating the issue directly on your app.

**Comment by kampalex** [comment]
Thanks for your answer. I've submit; ticket 522005.

**Comment by dmathieu** [comment]
This is fixed with #9 and #10.
